### DISCUSSION TITLE
Why developping own conponents when option like daisyUI could do the job ?

### DISCUSSION THREAD

**Discussion by pyladune** [petalframework/petal_components#206] [Q&A]
Hi,

What have been the reasons to choose de develop your own UI components when solition like daisyUI already interface pretty well tailwind css? Licensing ?

Thanks

**Comment by mplatts** [comment]
The main difference is our components are done in HEEX and thus can be more easily used in a Phoenix project. DaisyUI works fine, but you still need to take the extra step in converting them to HEEX components. We took inspiration from Flowbite, Tailwind UI and Flowbite.
